app/models/user.rb

```diff
@@ -26,6 +26,10 @@ def self.from_omniauth(auth)
     role, daac = Cmr.get_role_and_daac(auth.uid, auth.credentials['access_token'])
     user.role = role
     user.daac = daac if daac
+    auth.info['first_name'] = '' if auth.info['first_name'].nil?
+    auth.info['last_name'] = '' if auth.info['last_name'].nil?
+    user.name = auth.info['first_name']+' '+auth.info['last_name']
```

Comment:
I think rubocop should have yelled at you about this - you should use interpolation in double quotes here instead of using `+' '+` to concatenate strings.